themes/digital.gov/src/scss/new/_uswds-overrides.scss

```diff
@@ -56,3 +56,13 @@
     mask-image: url("../img/svg-icons/material-icon-inventory_2.svg");
   }
 }
+
+// overrides usa footer until 3.8.0 for uswds comes out and refactor footer into the uswds footer component.
+// turing off sylelint due to it flagging needing to be in BEM format
+/* stylelint-disable */
+@media (min-width: 30em) {
+  .usa-footer .grid-row .mobile-lg\:grid-col-6 {
```

Comment:
Based on the markup

```html
<!-- Footer column: Half on mobile and fourths on tablet-lg. -->
<div class="mobile-lg:grid-col-6 tablet-lg:grid-col-3">
```

We should be targeting `grid-col-3` because we want half columns on mobile-lg and then one fourth columns on `tablet-lg`.

```suggestion
  .usa-footer .grid-row .mobile-lg\:grid-col-3 {
```

Reply:
Shouldn't the grid selector read like `.mobile-lg:grid-col-3`.  The `\` looks to be copied from the browser which adds this to escape it. See https://designsystem.digital.gov/utilities/layout-grid/

Reply:
I believe the `\` is needed for the class naming... when it is removed it breaks and the issues is back with how the footer is looking. 

![Screenshot 2024-02-08 at 10 13 28 AM](https://github.com/GSA/digitalgov.gov/assets/88721460/85413f41-d884-40f9-991b-9da9dd0d8ec8)